mono/metadata/appdomain.c

```diff
@@ -131,9 +131,9 @@ get_shadow_assembly_location_base (MonoDomain *domain, MonoError *error);
 static MonoLoadFunc load_function = NULL;
 
 /* Lazy class loading functions */
-static GENERATE_GET_CLASS_WITH_CACHE (assembly, System.Reflection, Assembly);
+static GENERATE_GET_CLASS_WITH_CACHE (assembly, System.Reflection, "Assembly");
```

Comment:
There were a couple of places we used e.g. ``IEnumerable`1`` unquoted. These can just be strings because we immediately quoted them with `#name` anyway.